lib/sites/infibeam.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const siteUtils = require('../site-utils');
+const logger = require('../logger')();
+
+class InfibeamSite {
+  constructor(uri) {
+    if (!InfibeamSite.isSite(uri)) {
+      throw new Error('invalid uri for Infibeam: %s', uri);
+    }
+
+    this._uri = uri;
+  }
+
+  getURIForPageData() {
+    return this._uri;
+  }
+
+  isJSON() {
+    return false;
+  }
+
+  findPriceOnPage($) {
+    // the various ways we can find the price
+    const selectors = [
+      '#price-after-discount',
+    ];
+
+    // find the price on the page
+    let priceStorage = siteUtils.findContentOnPage($, selectors);
+    if (priceStorage !== null) {
+      priceStorage = priceStorage.substring(2, priceStorage.length);
+      priceStorage = priceStorage.replace(',', '');
+    }
```

Comment:
Get it. Since I don't plan on adding any further indian sites soon, I thought it'd be best left it at site instead of utils. Will change all these and put it in new pull request!